src/styles/blocks/card.scss

```
@@ -0,0 +1,93 @@
+.card {
+  width: 198px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding-top: 32px;
```

For example to the general message, according to [this](https://codeguide.academy/html-css.html#css-order) example, your .card properties order should be next:

```suggestion
.card {
  width: 198px;
  padding-top: 32px;
  
  background-color: #fff;
  border: 1px solid #f3f3f3;
  border-radius: 5px;
```